meshtastic/ble_interface.py

```
@@ -56,54 +150,244 @@ def __init__(
+                try:
+                    while True:
+                        if self._closing or not self.auto_reconnect:
+                            logger.debug(
+                                "Auto-reconnect aborted because interface is closing or disabled."
+                            )
+                            return
+                        try:
+                            logger.debug("Attempting BLE auto-reconnect.")
+                            self.connect(self.address)
+                            logger.info("BLE auto-reconnect succeeded.")
+                            return
```

Ruff (0.13.2)

Consider moving this statement to an `else` block

(TRY300)